packages/web-config-server/src/apiV1/dataBuilders/generic/count/countEvents.js

```diff
@@ -69,7 +69,7 @@ export class CountEventsBuilder extends DataBuilder {
     const { groupBy } = this.config;
     if (groupBy) {
       const hierarchyId = await this.fetchEntityHierarchyId();
-      groupEvents(this.models, events, { ...groupBy, hierarchyId });
+      return groupEvents(this.models, events, { ...groupBy, hierarchyId });
```

Comment:
The third great catch by unit tests

Reply:
Makes me wonder how many catches we'll miss because of no/wrong unit tests 😆

Reply:
But QA should catch those! 🙏

Reply:
Yes I'm nervous about the same thing! It'll be a big QA job...